docs/_sass/_header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  @include gradient-directional($black, $gray-darker, 165deg);
+  color: $black;
+  padding: 4rem 0;
+}
+
+.docs {
+  .header {
+    @include gradient-directional($black, $gray-darker, 165deg);
+    padding-bottom: 2rem;
+    padding-top: 2rem;
+
+    .title-container {
+      padding-bottom: 2rem;
+      padding-top: 2rem;
+    }
+
+    h1 {
+      text-align: left;
+    }
+
+    .subtitle {
+      font-weight: 400;
+      line-height: 1.2;
+      padding-bottom: 15px;
+      text-transform: none !important;
+    }
+
+    .card.github {
+      background: #e6e6e6;
```

Comment:
- Consider using $gray-lighter here
- Consider using border-radius()
- Use color: $black here instead of #000; remove duplicate color definition on line 37
- Border is defined twice with two different rules